crates/node/scripts/install.js

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
+
+/**
+ * @tailwindcss/oxide postinstall script
+ *
+ * This script ensures that the correct binary for the current platform and
+ * architecture is downloaded and available.
+ */
+
+const fs = require('fs')
+const path = require('path')
+const https = require('https')
+const { extract } = require('tar')
+const packageJson = require('../package.json')
+const detectLibc = require('detect-libc')
+
+const version = packageJson.version
+
+function getPlatformPackageName() {
```

Comment:
I wonder if we can somehow re-use this logic from what NAPI generated for us. Since that is generated and using require already it might not be as easy.

Just have to make sure that _if_ we make changes, that we keep this in sync